bigquery/README.md

```diff
@@ -2,50 +2,50 @@
-* If `~/.config/gcloud/application_default_credentials.json` doesn't exist, create it by running `gcloud auth application-default login`
-* `docker-compose up --build`
-* View Elasticsearch index:
- `http://localhost:9200/platinum_genomes/_search?pretty=true`
+* If `~/.config/gcloud/application_default_credentials.json` doesn't exist,
+create it by running `gcloud auth application-default login`.
+* Determine the project that will be billed for querying the BigQuery tables.
+Your account must have `bigquery.jobs.create` permissions this project; this
+includes any project where you have the Viewer/Editor/Owner role.
+* Run the indexer:
+  `BILLING_PROJECT_ID=<billing_project_id> docker-compose up --build`
+* View the Elasticsearch index at
+ `http://localhost:9200/platinum_genomes/_search?pretty=true`.
 
 If you want to run the Data Explorer UI on this dataset, follow the instructions
-below. Note that you will have to reindex the data into an Elasticsearch
+below. Note that you will have to re-index the data into an Elasticsearch
 container from the [data-explorer repo](https://github.com/DataBiosphere/data-explorer/).
 
 ### Index a custom dataset locally
 
-* If `~/.config/gcloud/application_default_credentials.json` doesn't exist, create it by running `gcloud auth application-default login`
+* If `~/.config/gcloud/application_default_credentials.json` doesn't exist,
+create it by running `gcloud auth application-default login`.
 * Setup config files.
   * Create `dataset_config/<my_dataset>`. Copy `dataset_config/template/*` to this directory.
   * Edit config files; instructions are in the files. Read
   [Overview](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery#overview)
   for some background information.
-* Run Elasticsearch.
-  * If you intend to run the Data Explorer UI from the [data-explorer repo](https://github.com/DataBiosphere/data-explorer/)
-    after this, run inside the data-explorer repo:
+* Run Elasticsearch:
+  * If you intend to run the [Data Explorer UI](https://github.com/DataBiosphere/data-explorer/)
+  after this, run inside the data-explorer repo:
     ```
     docker-compose up -d elasticsearch
     ```
   * If you do not intend to run the Data Explorer UI after this, and just want
-    to inspect the index in Elasticsearch, run inside this repo:
+  to inspect the index in Elasticsearch, run inside this repo:
     ```
     docker-compose up -d elasticsearch
     ```
+* Determine the project that will be billed for querying the BigQuery tables.
+Your account must have `bigquery.jobs.create` permissions this project; this
+includes any project where you have the Viewer/Editor/Owner role.
 * Run the indexer:
-`DATASET_CONFIG_DIR=dataset_config/<my dataset> docker-compose up --build indexer`
+  ```
+  BILLING_PROJECT_ID=<billing_project_id> DATASET_CONFIG_DIR=dataset_config/<my-dataset> docker-compose up --build indexer
```

Comment:
I realized that in data-explorer repo, I've been using <my dataset> with no spaces. Please update this whole PR.

Reply:
I'm not sure what this comment means

Reply:
Oops, forgot to put backticks around `<my dataset>`. For this whole PR, replace underscores and dashes with spaces, in `<billing_project_id>` and `<my-dataset>`.

Reply:
Done, spaces feels kind of weird, I think underscores are more conventional for this, but it doesn't really matter.